Assets/MixedRealityToolkit.Extensions/SceneTransitionService/Scripts/CameraFaderQuad.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.MixedReality.Toolkit.Extensions.SceneTransitions
     /// </summary>
     public class CameraFaderQuad : ICameraFader
     {
-        const string QuadMaterialShaderName = "UI/Unlit/Transparent";
+        const string QuadMaterialShaderName = "Sprites/Default";
```

Comment:
this may impact other consumers, is this necessary?

Reply:
@Railboy

Reply:
I think this was the fix for the issue with the Single Pass Instanced rendering, rendering on only one eye.

Reply:
@cre8ivepark Yes this change was related to the one-eye issue.

@Cameron-Micka At one point we were looking into using the MRTK standard shader, did that not pan out?